frontend/src/features/admin-form/create/featureTour/FeatureTour.tsx

```diff
@@ -0,0 +1,53 @@
+import Joyride from 'react-joyride'
+
+import { FEATURE_TOUR } from './constants'
+import { FeatureTourTooltip } from './FeatureTourTooltip'
+
+interface FeatureTourProps {
+  shouldRun: boolean
+}
+
+const featureTourSteps = [
+  {
+    target: `#${FEATURE_TOUR.FIRST_STEP.id}`,
+    title: `${FEATURE_TOUR.FIRST_STEP.title}`,
+    content: `${FEATURE_TOUR.FIRST_STEP.content}`,
+    disableBeacon: true,
+  },
+  {
+    target: `#${FEATURE_TOUR.SECOND_STEP.id}`,
+    title: `${FEATURE_TOUR.SECOND_STEP.title}`,
+    content: `${FEATURE_TOUR.SECOND_STEP.content}`,
+    disableBeacon: true,
+  },
+  {
+    target: `#${FEATURE_TOUR.THIRD_STEP.id}`,
+    title: `${FEATURE_TOUR.THIRD_STEP.title}`,
+    content: `${FEATURE_TOUR.THIRD_STEP.content}`,
+    disableBeacon: true,
+  },
+]
+
+export const FeatureTour = ({ shouldRun }: FeatureTourProps): JSX.Element => {
+  return (
+    <>
+      <Joyride
+        steps={featureTourSteps}
+        run={shouldRun}
+        hideBackButton
+        floaterProps={{
+          placement: 'right',
+          styles: {
+            arrow: {
+              length: 8,
+              spread: 16,
+            },
+          },
+        }}
+        spotlightPadding={3}
```

Comment:
3 seems to be a weird number, any reason? add comments?

Reply:
The react-joyride package has a prop to adjust the size of the spotlight on the targeted `DrawerIcon`. The default spotlight size is very big (size 10) and according to the Figma design, it seems to be much smaller than what the default spotlight size looks like. Hence I tried various numbers for the spotlight padding to adjust the size of the spotlight and it seems that 3 is the closest to the spotlight size in the Figma design.

Reply:
Attached below shows the type of spotlightPadding for the props that `Joyride` accepts.

<img width="458" alt="Screenshot 2022-05-30 at 5 21 47 PM" src="https://user-images.githubusercontent.com/48427064/170960974-87872013-213c-448f-b98c-f55aae8e5736.png">